hollow/src/main/java/com/netflix/hollow/api/producer/validation/DuplicateDataDetectionValidator.java

```diff
@@ -32,13 +32,14 @@
 
 /**
  * 
```

Comment:
Might be good idea to provide some javadoc on what this validator does.

Reply:
Ok.

Reply:
Done. Added some for also RecordCountValidator